### ISSUE TITLE
Precision adjustment of the coordinates puts some holes outside of their shell in ContourBuilder

### ISSUE THREAD

**Issue by gcheval** [graphhopper/graphhopper#2160]
**Describe the bug**
An exception is thrown when trying to find shells containing holes in ContourBuilder. 

Introduced in: 0e9461e6de94d491f2118fca20ca903f631e7288

**To Reproduce**

1. checkout **master**
2. download [new-york-latest.osm.pbf](https://download.geofabrik.de/north-america/us/new-york-latest.osm.pbf)
3. Use any GTFS file, as long a PT isochronecan be generated without PT data interfering.
4. Query: `curl --location --request GET 'http://localhost:8989/isochrone-pt?point=40.744823,-73.991453&time_limit=4000&pt.earliest_departure_time=2000-11-01T08:00:00Z'`
5. See error (Found a hole without a shell.)

Here's a smaller polygon with the same issue: `http://localhost:8989/isochrone-pt?point=40.705805,-74.017997&time_limit=200&pt.earliest_departure_time=2000-11-01T08:00:00Z`

**Screenshots & Logs**

The Exception:
```
java.lang.RuntimeException: Found a hole without a shell.
	at com.graphhopper.isochrone.algorithm.ContourBuilder.punchHoles(ContourBuilder.java:148)
	at com.graphhopper.isochrone.algorithm.ContourBuilder.computeIsoline(ContourBuilder.java:104)
	at com.graphhopper.resources.PtIsochroneResource.doGet(PtIsochroneResource.java:166)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)
	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)
	at io.dropwizard.jetty.NonblockingServletHolder.handle(NonblockingServletHolder.java:50)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1631)
	at io.dropwizard.servlets.ThreadNameFilter.doFilter(ThreadNameFilter.java:35)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1618)
	at io.dropwizard.jersey.filter.AllowedMethodsFilter.handle(AllowedMethodsFilter.java:47)
	at io.dropwizard.jersey.filter.AllowedMethodsFilter.doFilter(AllowedMethodsFilter.java:41)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1618)
	at com.graphhopper.http.CORSFilter.doFilter(CORSFilter.java:36)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1618)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:549)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1363)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:489)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1278)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at com.codahale.metrics.jetty9.InstrumentedHandler.handle(InstrumentedHandler.java:249)
	at io.dropwizard.jetty.RoutingHandler.handle(RoutingHandler.java:52)
	at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:767)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:173)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:500)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:547)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.lang.Thread.run(Thread.java:748)
```
I created a Multipolygon containing the hole (the first polygon) and its shell

The Wicket screenshots:
![Screen Shot 2020-10-23 at 9 44 33 AM](https://user-images.githubusercontent.com/3153929/97011320-86cc5780-1514-11eb-9304-c500ed0027e8.png)
![Screen Shot 2020-10-23 at 9 44 44 AM](https://user-images.githubusercontent.com/3153929/97011365-92b81980-1514-11eb-9ff2-39960d99bb68.png)

The multipolygon:
```
MULTIPOLYGON(((-74.0189079 40.70492627,-74.01887623 40.70493074,-74.01887623 40.70493074,-74.01879688 40.70499835,-74.01871046 40.70491509,-74.01869146 40.70487598,-74.01869146 40.70487598,-74.01864955 40.70473311,-74.01866818 40.70470964,-74.0189079 40.70492627)),((-74.03213269 40.71381573,-74.01923239 40.70572142,-74.01890792 40.70492626,-74.0189079 40.70492627,-74.01866817 40.70470964,-74.01829434 40.70442428,-74.01829434 40.70442428,-74.01784023 40.70425069,-74.01764428 40.70389082,-74.01765359 40.7038016,-74.01765434 40.70373399,-74.01764502 40.70363527,-74.01763981 40.70358591,-74.01763981 40.70358591,-74.01762193 40.70349892,-74.01758337 40.70337748,-74.01754072 40.7032674,-74.01746435 40.70311298,-74.01746437 40.70311298,-74.01746435 40.70311297,-74.01723114 40.70283935,-74.01690742 40.70246868,-74.01686849 40.70242715,-74.01652501 40.70207082,-74.01648497 40.70202929,-74.01614075 40.70168172,-74.01607556 40.70162956,-74.0157062 40.70132204,-74.0157062 40.70132205,-74.01567453 40.70134477,-74.01567453 40.70134477,-74.01560003 40.70133974,-74.01555067 40.7013254,-74.01538582 40.70124848,-74.01538582 40.70124848,-74.01514666 40.70088787,-74.01514666 40.70088786,-74.01405478 40.70085061,-74.01302585 40.70054904,-74.01161043 40.70080348,-74.00935905 40.70161001,-74.00935904 40.70161001,-74.0055002 40.70243013,-74.0055002 40.70243013,-74.00220052 40.70564413,-74.00220052 40.70564413,-74.00143759 40.70605279,-73.99824203 40.70718379,-73.99824203 40.70718379,-73.99824203 40.70718379,-73.9944849 40.70420915,-73.99448489 40.70420915,-73.99424294 40.70402214,-73.99424294 40.70402214,-73.99424294 40.70402214,-73.99424294 40.70402214,-73.99448489 40.70420915,-73.99448489 40.70420915,-73.9941282 40.70929156,-73.99400023 40.70929938,-73.99385178 40.70930963,-73.99182019 40.70958288,-73.99182019 40.70958288,-73.9897331 40.70988258,-73.9897331 40.70988258,-73.98818133 40.70919936,-73.98818133 40.70919936,-73.98818133 40.70919936,-73.98818133 40.70919936,-73.98665675 40.7098822,-73.98646956 40.70990176,-73.98421464 40.70958753,-73.98421464 40.70958753,-73.98363797 40.70962013,-73.98363796 40.70962013,-73.98363796 40.70962013,-73.98363796 40.70962014,-73.98246599 40.71080291,-73.98234659 40.71087649,-73.98200443 40.71142764,-73.98185299 40.71143808,-73.98144992 40.7114662,-73.98056348 40.71106573,-73.98056348 40.71106573,-73.98039976 40.71071109,-73.98039976 40.71071109,-73.98039976 40.71071108,-73.97924287 40.71014465,-73.97924287 40.71014465,-73.97865204 40.71051402,-73.97772518 40.71077553,-73.97732714 40.71106853,-73.97685403 40.71146378,-73.97685402 40.71146378,-73.97637495 40.71274211,-73.97219574 40.71354025,-73.97219573 40.71354025,-73.97219573 40.71354025,-73.97219573 40.71354026,-73.97213706 40.7136509,-73.97213706 40.7136509,-73.97213706 40.7136509,-73.97213706 40.7136509,-73.97422471 40.71714429,-73.97422471 40.71714429,-73.97422471 40.71714429,-73.97317809 40.72007963,-73.97315835 40.72015935,-73.97296463 40.72094166,-73.97294247 40.72103144,-73.97294246 40.72103144,-73.97233673 40.72364026,-73.97233673 40.72364026,-73.97233673 40.72364026,-73.97224099 40.72407109,-73.97208118 40.72591846,-73.97160285 40.72823186,-73.96250738 40.72975588,-73.96250737 40.72975588,-73.96250737 40.72975588,-73.96250737 40.72975589,-73.96250737 40.72975589,-73.96250737 40.72975589,-73.9612974 40.73096884,-73.96118918 40.73026979,-73.96117763 40.73015617,-73.96117763 40.73015617,-73.96116906 40.73009358,-73.9610981 40.72991309,-73.9610981 40.72991309,-73.96100441 40.72966294,-73.96100441 40.72966294,-73.96099155 40.7295441,-73.96099062 40.7294899,-73.96099062 40.7294899,-73.96082913 40.72899276,-73.96078182 40.72861055,-73.96078182 40.72861055,-73.96075388 40.72814917,-73.96054098 40.72779657,-73.96054098 40.72779657,-73.95915052 40.72778912,-73.95877687 40.72689095,-73.95877687 40.72689095,-73.95877687 40.72689095,-73.95877687 40.72689095,-73.958672 40.72785413,-73.95724298 40.72775392,-73.95724298 40.72775392,-73.95724298 40.72775392,-73.95724298 40.72775392,-73.95735995 40.72845799,-73.95735995 40.728458,-73.95720088 40.72983896,-73.95720088 40.72983896,-73.95468296 40.73020087,-73.954574 40.730088,-73.954159 40.72983188,-73.9539569 40.72950294,-73.9539569 40.72950294,-73.953066 40.73024893,-73.953066 40.73024893,-73.95250683 40.73035994,-73.95156694 40.72869772,-73.95156694 40.72869772,-73.95156694 40.72869772,-73.95072223 40.72897637,-73.95072223 40.72897637,-73.95059296 40.72901902,-73.95059296 40.72901902,-73.95043371 40.72916058,-73.94991086 40.72924384,-73.94991086 40.72924385,-73.94991086 40.72924385,-73.94991086 40.72924385,-73.94932171 40.72990881,-73.94932171 40.72990881,-73.94876999 40.73072185,-73.94835388 40.73068386,-73.94835388 40.73068386,-73.94789492 40.7308839,-73.94789492 40.73088391,-73.94789492 40.73088391,-73.94789492 40.73088391,-73.94789492 40.73088391,-73.94789492 40.73088391,-73.94789492 40.73088391,-73.94933791 40.73138682,-73.94945097 40.73209388,-73.94945098 40.73209388,-73.94945098 40.73209388,-73.94957298 40.73280094,-73.94957298 40.73280094,-73.94968399 40.73350893,-73.94968399 40.73350893,-73.94980283 40.73422381,-73.94992483 40.7349359,-73.94864967 40.73609577,-73.94788785 40.73592385,-73.94788784 40.73592385,-73.94718954 40.73544683,-73.94718954 40.73544683,-73.94650855 40.73674025,-73.94650855 40.73674025,-73.94650855 40.73674025,-73.94598645 40.73910394,-73.94529225 40.73955526,-73.94299747 40.73911232,-73.94248785 40.73856527,-73.94183052 40.73757937,-73.94168319 40.73735082,-73.94168319 40.73735082,-73.94168319 40.73735082,-73.94168318 40.73735082,-73.94137864 40.73747078,-73.94019009 40.73751753,-73.93895404 40.73559807,-73.93876666 40.73528589,-73.93876666 40.73528589,-73.93855413 40.73494633,-73.93855413 40.73494633,-73.93855413 40.73494633,-73.93855413 40.73494633,-73.93828349 40.73534252,-73.93824139 40.73560143,-73.93824139 40.73560143,-73.93824139 40.73560143,-73.93836563 40.73580725,-73.93770998 40.73722323,-73.93751999 40.73692297,-73.93750658 40.73613675,-73.93750658 40.73613675,-73.93750658 40.73613675,-73.93730839 40.73621405,-73.93721079 40.73624627,-73.93711114 40.73628129,-73.93711114 40.73628129,-73.93711114 40.73628129,-73.93711114 40.73628129,-73.93603285 40.73748586,-73.93603285 40.73748586,-73.93542135 40.73754938,-73.93542135 40.73754938,-73.93542135 40.73754938,-73.93542135 40.73754938,-73.93582815 40.73792619,-73.93582815 40.73792619,-73.93664603 40.73862264,-73.93664603 40.73862264,-73.93664604 40.73862264,-73.93688334 40.73877444,-73.93688334 40.73877444,-73.93690737 40.73865039,-73.93773047 40.73818752,-73.93863012 40.73837975,-73.93874822 40.73869193,-73.93853103 40.74041059,-73.93819427 40.74054228,-73.93726946 40.74046405,-73.93726946 40.74046405,-73.93631206 40.74033739,-73.93631206 40.74033739,-73.93631206 40.74033739,-73.93613697 40.74123518,-73.93577711 40.74308274,-73.93486889 40.74297806,-73.93486888 40.74297806,-73.93486888 40.74297806,-73.93394334 40.74287152,-73.93394334 40.74287152,-73.93394334 40.74287152,-73.9337703 40.74370356,-73.93284847 40.74359758,-73.93284847 40.74359758,-73.93171599 40.74451139,-73.93171599 40.74451139,-73.93078932 40.74440019,-73.93078932 40.74440019,-73.93078932 40.74440019,-73.93078932 40.74440019,-73.93078932 40.74440019,-73.93159845 40.74466785,-73.93158038 40.74475633,-73.93158038 40.74475633,-73.93164688 40.74483549,-73.93133209 40.74641706,-73.93133209 40.74641706,-73.93128869 40.74668584,-73.93128869 40.74668584,-73.93128869 40.74668584,-73.9312887 40.74668584,-73.9312887 40.74668584,-73.93206002 40.74662698,-73.93227292 40.74648337,-73.93326738 40.74627848,-73.9343859 40.74618814,-73.9343859 40.74618814,-73.93454497 40.74615108,-73.93638023 40.74545855,-73.93679225 40.74833242,-73.93665851 40.74841587,-73.935388 40.74940996,-73.935388 40.74940996,-73.935388 40.74940996,-73.93474297 40.75097887,-73.93286356 40.75076634,-73.93286356 40.75076634,-73.93251078 40.7510692,-73.93251078 40.75106921,-73.93284736 40.75110068,-73.93283134 40.75121654,-73.93287958 40.75132513,-73.93264191 40.75190888,-73.93264191 40.75190889,-73.93264191 40.75190889,-73.93264191 40.75190889,-73.93275273 40.75188765,-73.93275273 40.75188765,-73.93342012 40.75171182,-73.93351865 40.75192397,-73.93351865 40.75192397,-73.93475675 40.75196532,-73.93475675 40.75196532,-73.93475675 40.75196533,-73.93561394 40.75228197,-73.93561394 40.75228197,-73.93640184 40.75267183,-73.93600882 40.75434094,-73.93600882 40.75434094,-73.93600882 40.75434094,-73.93600883 40.75434094,-73.93721377 40.75379463,-73.9375185 40.75395575,-73.93754886 40.75515995,-73.93754886 40.75515995,-73.93754886 40.75515995,-73.93829392 40.75555688,-73.93848074 40.75663796,-73.93811883 40.75703097,-73.93811883 40.75703097,-73.93811883 40.75703097,-73.93811883 40.75703097,-73.93859995 40.75800644,-73.93859995 40.75800644,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93854016 40.75824505,-73.93863106 40.75814893,-73.93870519 40.75808318,-73.93991256 40.75786022,-73.93991256 40.75786022,-73.93991256 40.75786022,-73.93991256 40.75786022,-73.93991256 40.75786022,-73.93993398 40.75786954,-73.93998352 40.75789096,-73.94003288 40.75790567,-73.94003288 40.75790567,-73.94003288 40.75790567,-73.94003288 40.75790567,-73.94086195 40.75811094,-73.94086195 40.75811094,-73.94086195 40.75811094,-73.94086195 40.75811094,-73.94166195 40.75839797,-73.94166195 40.75839797,-73.94166195 40.75839797,-73.94166195 40.75839797,-73.94166195 40.75839797,-73.94159564 40.75997023,-73.94156156 40.76003244,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94151927 40.76010937,-73.94157906 40.76013284,-73.94165879 40.76016413,-73.94165879 40.76016413,-73.94233604 40.76040087,-73.94233604 40.76040087,-73.94233604 40.76040087,-73.94239732 40.76042583,-73.94245805 40.7604506,-73.94242452 40.76188782,-73.94241651 40.76190086,-73.94237833 40.7619627,-73.94233772 40.76202845,-73.94232934 40.76204223,-73.94232934 40.76204223,-73.94232934 40.76204223,-73.94232934 40.76204224,-73.94232934 40.76204223,-73.94241539 40.76208955,-73.94241539 40.76208955,-73.94251039 40.76209048,-73.94252715 40.76212847,-73.94252715 40.76212847,-73.94253516 40.76211674,-73.94254652 40.76210016,-73.942559 40.76208154,-73.94260184 40.76201709,-73.94264208 40.76195506,-73.94265269 40.76193886,-73.94266163 40.76191874,-73.94267095 40.76189918,-73.94267095 40.76189918,-73.94268343 40.76187273,-73.94313475 40.76180512,-73.94348958 40.76177532,-73.94348958 40.76177532,-73.94348958 40.76177532,-73.94354583 40.76159054,-73.94388297 40.76114593,-73.94454421 40.76138155,-73.94454421 40.76138155,-73.94454421 40.76138155,-73.94518868 40.76135864,-73.94518869 40.76135864,-73.94548503 40.76077489,-73.94582031 40.76060018,-73.94631111 40.76067841,-73.94798079 40.76243711,-73.94725808 40.76329058,-73.94725808 40.76329058,-73.94725808 40.76329058,-73.94725808 40.76329058,-73.9470761 40.76376444,-73.9470761 40.76376444,-73.94683787 40.76414497,-73.94671717 40.76416323,-73.94666725 40.76397789,-73.94675759 40.76385328,-73.94675759 40.76385328,-73.94675759 40.76385328,-73.94665477 40.76397138,-73.94645193 40.76386465,-73.94645193 40.76386465,-73.94645193 40.76386465,-73.94635712 40.76432602,-73.94583353 40.76494498,-73.94579889 40.76498614,-73.94579889 40.76498614,-73.94579888 40.76498615,-73.94579889 40.76498615,-73.94579889 40.76498615,-73.94579889 40.76498615,-73.94595367 40.76608622,-73.94595367 40.76608622,-73.94595367 40.76608622,-73.94610194 40.76591691,-73.94669594 40.76537767,-73.94671717 40.76538848,-73.94676374 40.76541176,-73.94700365 40.76553339,-73.94712043 40.76559262,-73.94714018 40.76560175,-73.94714018 40.76560175,-73.94818345 40.7652648,-73.94818345 40.7652648,-73.94818345 40.7652648,-73.94831607 40.76509418,-73.94831607 40.76509418,-73.94856101 40.7648023,-73.94873181 40.76455904,-73.94919375 40.76405091,-73.95156582 40.7654749,-73.9510318 40.76651128,-73.9510318 40.76651128,-73.9510318 40.76651128,-73.95140247 40.76674504,-73.95153788 40.76680353,-73.95153788 40.76680353,-73.95223134 40.76870454,-73.95218683 40.76876638,-73.95214995 40.76881742,-73.95214995 40.76881742,-73.95214995 40.76881742,-73.95190613 40.76940024,-73.95190613 40.76940024,-73.95190613 40.76940024,-73.95202199 40.76944904,-73.95354581 40.77091271,-73.95354581 40.77091271,-73.95351154 40.77095927,-73.95346069 40.77102838,-73.95346069 40.77102838,-73.95307959 40.77154545,-73.95307959 40.77154545,-73.95307959 40.77154545,-73.95304532 40.77159555,-73.95299969 40.77166224,-73.95299969 40.77166224,-73.95299969 40.77166224,-73.95299969 40.77166224,-73.95303135 40.77167509,-73.95313939 40.77171905,-73.953245 40.77176133,-73.95327908 40.77177493,-73.95327908 40.77177493,-73.95327908 40.77177493,-73.95496571 40.77330528,-73.95447416 40.77397955,-73.95447416 40.77397956,-73.95413925 40.77460969,-73.95413925 40.77460969,-73.95409939 40.77466594,-73.95409939 40.77466594,-73.95421823 40.774714,-73.95421823 40.774714,-73.95586946 40.77623484,-73.95540641 40.77686945,-73.95540641 40.77686945,-73.95540641 40.77686945,-73.95540641 40.77686945,-73.95654505 40.77818112,-73.9560833 40.77881219,-73.9560833 40.77881219,-73.9560833 40.77881219,-73.9560833 40.77881219,-73.95707553 40.78000204,-73.95700307 40.78010076,-73.95693266 40.78019669,-73.95693266 40.78019669,-73.95693266 40.78019669,-73.95655194 40.78071655,-73.95655194 40.78071656,-73.95651115 40.78077243,-73.95651115 40.78077244,-73.95656013 40.7808719,-73.95656013 40.7808719,-73.95656013 40.7808719,-73.95663892 40.78090506,-73.95663892 40.78090506,-73.95670449 40.78093262,-73.95670449 40.78093262,-73.95677359 40.78096094,-73.95687008 40.78100229,-73.95779302 40.78206139,-73.95775111 40.78211969,-73.95770752 40.78217929,-73.95770752 40.78217929,-73.95770752 40.78217929,-73.95733741 40.7826904,-73.95733741 40.7826904,-73.957297 40.78274647,-73.9572536 40.78280644,-73.9572536 40.78280644,-73.9572536 40.78280644,-73.95736033 40.78285152,-73.95747469 40.78289995,-73.95747469 40.78289995,-73.9584971 40.78399705,-73.95845575 40.78405479,-73.95841458 40.78411197,-73.95841458 40.78411197,-73.95804205 40.78463053,-73.95800238 40.78467915,-73.95800238 40.78467915,-73.95805118 40.7847788,-73.95805118 40.7847788,-73.95805118 40.7847788,-73.9580957 40.78471789,-73.95825086 40.78471417,-73.95825086 40.78471417,-73.95891806 40.78457372,-73.95902702 40.78442639,-73.95917622 40.78422299,-73.95972179 40.78380296,-73.95972179 40.78380296,-73.96508769 40.78414196,-73.9650877 40.78414196,-73.96567331 40.78809207,-73.96567331 40.78809207,-73.96521063 40.78901017,-73.96521063 40.78901017,-73.96521063 40.78901017,-73.96589273 40.78949744,-73.96593967 40.78973399,-73.96593967 40.78973399,-73.96593967 40.78973399,-73.96605739 40.78978969,-73.96605739 40.78978969,-73.96618293 40.78984948,-73.96618293 40.78984948,-73.96618293 40.78984948,-73.96622335 40.78979435,-73.96626452 40.78973791,-73.96626452 40.78973791,-73.96664003 40.78922475,-73.96668007 40.78917017,-73.96671975 40.78911597,-73.96671975 40.78911597,-73.96709805 40.7885989,-73.96709805 40.7885989,-73.9671489 40.78852905,-73.96718206 40.78848379,-73.96987469 40.78968463,-73.9698747 40.78968464,-73.97015074 40.78930149,-73.97268617 40.79086835,-73.97268617 40.79086835,-73.97278526 40.79091007,-73.97278526 40.79091007,-73.97278526 40.79091007,-73.97324236 40.79028366,-73.97449629 40.79080595,-73.97449629 40.79080595,-73.97467957 40.79087412,-73.97467958 40.79087412,-73.97513182 40.79025516,-73.97645635 40.79081731,-73.97645635 40.79081731,-73.97668974 40.79049563,-73.97692313 40.79017414,-73.97692313 40.79017414,-73.97737426 40.78956263,-73.97737426 40.78956263,-73.97961931 40.79098308,-73.97961931 40.79098308,-73.97961931 40.79098308,-73.97987319 40.79061726,-73.98059999 40.79074243,-73.98075049 40.79078657,-73.98075049 40.79078657,-73.98075049 40.79078658,-73.98075049 40.79078657,-73.98114835 40.79000501,-73.98194855 40.78866428,-73.98306501 40.7887779,-73.98590927 40.78647567,-74.01045574 40.7765597,-74.01045574 40.77655971,-74.01045408 40.77661043,-74.01045578 40.77655968,-74.01045577 40.77655968,-74.01045576 40.77655968,-74.01045576 40.77655968,-74.01045576 40.77655969,-74.01045576 40.77655969,-73.99057446 40.77969526,-73.99202285 40.77605286,-73.99233839 40.77559074,-73.99279883 40.77461509,-73.99291618 40.77444112,-73.99302719 40.77427683,-73.99328964 40.77389667,-73.99368321 40.77343436,-73.99392964 40.77305494,-73.99426175 40.77234527,-73.99477212 40.7717116,-73.99477211 40.7717116,-73.99490194 40.77151434,-73.99497552 40.77142084,-73.99533724 40.7710742,-73.99536891 40.77103006,-73.99574013 40.7711867,-73.99574013 40.77118671,-73.99600574 40.77046996,-73.99603015 40.77014903,-73.99603015 40.77014902,-73.99663588 40.76828098,-73.99728222 40.76695925,-73.99728222 40.76695924,-73.99734629 40.76620692,-73.99751039 40.76598191,-73.99780543 40.76557772,-73.99796692 40.76564105,-73.99808874 40.76572729,-73.99818318 40.76572561,-73.99818318 40.76572562,-73.99818318 40.76572561,-73.99847114 40.76527262,-73.99847114 40.76527262,-73.99852236 40.76500607,-73.99830853 40.76488854,-73.99871608 40.76433031,-73.99889918 40.76449292,-73.99900516 40.76433534,-73.99900516 40.76433534,-73.99900516 40.76433534,-73.99878258 40.76423904,-73.9987036 40.76394753,-73.9987036 40.76394753,-73.99896884 40.76361766,-73.9997912 40.76315144,-73.99991916 40.7632293,-73.99991916 40.7632293,-74.00010543 40.76321384,-74.00114776 40.7629933,-74.00351593 40.76361096,-74.00351593 40.76361096,-74.00351594 40.76361095,-74.00425298 40.76247511,-74.00425298 40.76247511,-74.00442136 40.76034592,-74.00442136 40.76034592,-74.00416189 40.76023044,-74.0040859 40.76019933,-74.00375751 40.76005274,-74.00375751 40.76005274,-74.003751 40.75923001,-74.00404716 40.75901879,-74.00404716 40.75901879,-74.00432395 40.758479,-74.00530574 40.75702706,-74.00530575 40.75702707,-74.007152 40.75469839,-74.007152 40.75469838,-74.00742208 40.75401162,-74.00742208 40.75401162,-74.00750628 40.753795,-74.00750628 40.753795,-74.00780355 40.75234754,-74.00855383 40.75199326,-74.00859965 40.75195582,-74.00859965 40.75195582,-74.00855494 40.75174628,-74.00874568 40.75124914,-74.00965279 40.75090473,-74.00965279 40.75090474,-74.0107741 40.74863846,-74.02252533 40.75290019,-74.02281832 40.75879228,-74.02281832 40.75879229,-74.02281833 40.7587923,-74.02281839 40.75879222,-74.02281835 40.75879226,-74.02252535 40.75290016,-74.02655909 40.73503667,-74.02655909 40.73503666,-74.02655908 40.73503667,-74.02655908 40.73503667,-74.01389907 40.72899388,-74.01389906 40.72899388,-74.01389906 40.72899388,-74.01299493 40.72861669,-74.01299493 40.72861669,-74.01299493 40.72861669,-74.01203269 40.72883406,-74.01203269 40.72883406,-74.0113247 40.72877334,-74.01134109 40.72822926,-74.01132358 40.72810018,-74.01135226 40.72790107,-74.01139883 40.72770083,-74.01139883 40.72770083,-74.01139883 40.72770083,-74.01108684 40.7264726,-74.01108684 40.7264726,-74.01120139 40.72572233,-74.01170598 40.72545374,-74.01170598 40.72545374,-74.01170598 40.72545374,-74.01172964 40.72533919,-74.0117492 40.72524475,-74.01176894 40.72514882,-74.01189262 40.72445182,-74.01190119 40.72439892,-74.01190119 40.72439892,-74.01205988 40.72359854,-74.01218785 40.72295202,-74.01226347 40.72261413,-74.01229253 40.72248691,-74.01329538 40.72151871,-74.01329538 40.72151871,-74.01310445 40.72149655,-74.01313835 40.72105193,-74.01313835 40.72105193,-74.01313836 40.72105193,-74.0136612 40.72037579,-74.01399666 40.72013663,-74.01454987 40.72020219,-74.01449697 40.72047824,-74.01449697 40.72047824,-74.01512281 40.72055517,-74.01512282 40.72055516,-74.01512282 40.72055516,-74.01570657 40.72033947,-74.01606587 40.71857443,-74.01606587 40.71857443,-74.01675673 40.71817508,-74.01675673 40.71817509,-74.01802389 40.71525763,-74.03213267 40.71381576,-74.03213268 40.71381575,-74.03213269 40.71381574,-74.03213269 40.71381573)))
```

When I remove the precision modification to the coordinates, the hole is contained in the shell:
![Screen Shot 2020-10-23 at 9 52 56 AM](https://user-images.githubusercontent.com/3153929/97012539-f0993100-1515-11eb-8a4a-85e229ce86d2.png)

**Comment by gcheval** [comment]
Potential PR: 2161

**Comment by michaz** [comment]
Fixed by #2161
